src/pages/workspaces/workspace/Data.js

```diff
@@ -1028,7 +1029,9 @@ const WorkspaceData = _.flow(
                 setUploadingWDSFile(false)
                 forceRefresh()
                 loadMetadata()
-              }, namespace, name,
+                notify('success', 'Data imported successfully.')
```

Comment:
how difficult would it be to add the table name to this notification, e.g. `Data imported successfully to table "participant"'?

Reply:
Should be very easy